Example/RxRealm_Tests/TestModels.swift

```diff
@@ -24,15 +24,7 @@ class Message: Object {
     }
 }
 
-extension Array where Element: Message {
-    func equalTo(to: [Message]) -> Bool {
-        guard count == to.count else {return false}
-        let (result, _) = reduce((true, 0)) {acc, el in
-            guard acc.0 && self[acc.1] == to[acc.1] else {return (false, 0)}
-            return (true, acc.1+1)
-        }
-        return result
-    }
+extension Message: Equatable {
 }
```

Comment:
Yep the tests are passing on my local machine =)
As far, as I understand in Swift we can compare `Array`s if the elements in the `Array` are `Equatable`